tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/LabelHelper.cs

```diff
@@ -0,0 +1,99 @@
+using Azure.Sdk.Tools.Cli.Services;
+using System.Text;
+using CsvHelper;
+using CsvHelper.Configuration;
+
+namespace Azure.Sdk.Tools.Cli.Helpers
+{
+    public interface ILabelHelper
+    {
+        public LabelHelper.ServiceLabelStatus CheckServiceLabel(string csvContent, string serviceName);
+        public string CreateServiceLabel(string csvContent, string serviceLabel);
+        public string NormalizeLabel(string label);
+    }
+
+    public class LabelHelper(ILogger<LabelHelper> logger) : ILabelHelper
+    {
+        internal const string ServiceLabelColorCode = "e99695"; // color code for service labels in common-labels.csv
+
+        public static IList<LabelData> GetLabelsFromCsv(string csvContent)
+        {
+            using var reader = new StringReader(csvContent);
+            using var csvReader = new CsvReader(reader, config);
+            return csvReader.GetRecords<LabelData>().ToList();
+        }
+
+        private static CsvConfiguration config = new CsvConfiguration(System.Globalization.CultureInfo.InvariantCulture)
+        {
+            HasHeaderRecord = false,
+            NewLine = "\n",
+        };
+
+        public enum ServiceLabelStatus
+        {
+            Exists,
+            DoesNotExist,
+            NotAServiceLabel
+        }
+
+        public ServiceLabelStatus CheckServiceLabel(string csvContent, string serviceName)
+        {
+            var records = GetLabelsFromCsv(csvContent);
+
+            foreach (var record in records)
+            {
+                if (record.Name.Equals(serviceName, StringComparison.OrdinalIgnoreCase))
+                {
+                    if (record.Color.Equals(ServiceLabelColorCode, StringComparison.OrdinalIgnoreCase))
+                    {
+                        logger.LogInformation($"Service label '{serviceName}' exists in common-labels.csv.");
+                        return ServiceLabelStatus.Exists;
+                    }
+                    else
+                    {
+                        logger.LogWarning($"Label '{serviceName}' exists but is not a service label.");
+                        return ServiceLabelStatus.NotAServiceLabel;
+                    }
+                }
+
+            }
+
+            return ServiceLabelStatus.DoesNotExist;
+        }
+
+        public string CreateServiceLabel(string csvContent, string serviceLabel)
+        {
+            IList<LabelData> records;
```

Comment:
Similar to my above comment, it should be simple enough to update the csv content without `CsvHelper`.